src/main/java/ir/sahab/hbaserule/Builder.java

```diff
@@ -0,0 +1,74 @@
+package ir.sahab.hbaserule;
+
+import org.apache.hadoop.hbase.TableName;
+import org.apache.hadoop.hbase.util.Bytes;
+
+import java.util.Arrays;
+import java.util.Map;
+
+/**
+ * Builder class for building {@link HbaseExtension} and {@link HbaseRule}
+ *
+ * @param <T> {@link HbaseExtension} or {@link HbaseRule}
+ */
+class Builder<T extends HbaseBase> {
+
```

Comment:
Remove extra empty line. Please check these type of rule (this is very annoying)

Reply:
I added a checkstyle rule for this in my IDE.